locales/th-TH/translation.json

```
@@ -92,65 +92,65 @@
+        "last-day-balance": "วันสุดท้ายที่ได้ใช้ชั่วโมงส่วนเกิน",
         "on": "On",
-        "working-days": "Working days",
-        "working-days-title": "How many working days there's in the month",
-        "month-balance": "Month Balance",
-        "month-balance-title": "Balance up until today for this month. A positive balance means extra hours you don't need to work today (or the rest of the month).",
-        "overall-balance": "Overall Balance",
-        "overall-balance-title": "Overall balance until end of the month or current day",
-        "add-waiver-day": "Add a waiver for this day",
+        "working-days": "วันทำงาน",
+        "working-days-title": "จำนวนวันทำงานในเดือน",
+        "month-balance": "จำนวนชั่วโมงส่วนเกินในเดือนนี้",
+        "month-balance-title": "จำนวนชั่วโมงส่วนเกินจนถึงวันนี้สำหรับเดือนนี้ จำนวนที่เป็นบวกหมายถึงชั่วโมงส่วนเกินที่คุณไม่ต้องทำงานสำหรับวันนี้ (หรือสำหรับวันที่เหลือของเดือน)",
+        "overall-balance": "จำนวนชั่วโมงส่วนเกินทั้งหมด",
+        "overall-balance-title": "จำนวนชั่วโมงส่วนเกินทั้งหมด นับถึงวันปัจจุบันหรือสิ้นเดือนนี้",
+        "add-waiver-day": "เพิ่มวันหยุดสำหรับวันนี้",
         "waived-day": "วันหยุด",
-        "remove-entry": "Remove entry",
-        "entry-removal-confirmation": "Are you sure you want to remove the last two entries from day",
-        "yes": "Yes",
-        "no": "No"
+        "remove-entry": "ลบรายการ",
+        "entry-removal-confirmation": "คุณมั่นใจหรือไม่ที่จะลบสองรายการสุดท้ายจากวัน",
+        "yes": "ใช่",
+        "no": "ไม่"
     },
     "$Calendar": {
         "leave-by": "วันนี้ควรเลิกงาน",
-        "all-done": "All done for today. Balance of the day:",
-        "last-day-balance": "Last day used for balance",
+        "all-done": "วันนี้จบแล้ว. จำนวนชั่วโมงส่วนเกินของวันนี้:",
+        "last-day-balance": "วันสุดท้ายที่ได้ใช้ชั่วโมงส่วนเกิน",
         "on": "เมื่อวันที่",
         "working-days": "จำนวนวันทำงาน",
         "working-days-title": "จำนวนวันทำงานในแต่ละเดือน",
         "month-balance": "จำนวนชั่วโมงส่วนเกินในเดือนนี้",
-        "month-balance-title": "จำนวนชั่วโมงส่วนเกินในเดือนนี้. ถ้าค่าเป็นบวกแสดงว่าทำเกินมา ถ้าค่าติดลบแสดงว่าทำขาดไป.",
+        "month-balance-title": "จำนวนชั่วโมงส่วนเกินในเดือนนี้ ถ้าค่าเป็นบวกแสดงว่าทำเกินมา ถ้าค่าติดลบแสดงว่าทำขาดไป",
         "overall-balance": "จำนวนชั่วโมงส่วนเกินทั้งหมด",
-        "overall-balance-title": "จำนวนชั่วโมงส่วนเกินทั้งหมด. นับถึงวันปัจจุบันหรือสิ้นเดือนนี้.",
+        "overall-balance-title": "จำนวนชั่วโมงส่วนเกินทั้งหมด นับถึงวันปัจจุบันหรือสิ้นเดือนนี้",
         "switch-view": "เปลี่ยนมุมมอง",
         "switch-view-title": "เปลี่ยนมุมมอง",
         "current-month": "เดือนปัจจุบัน",
         "current-month-title": "ไปที่เดือนปัจจุบัน",
         "previous-month": "เดือนที่แล้ว",
         "next-month": "เดือนถัดไป",
-        "time-to-leave": "Time to Leave",
+        "time-to-leave": "เวลาเลิกงาน",
```

Time to Leave is the name of the App, because of this we are not translating it. :)
```suggestion
        "time-to-leave": "Time to Leave",
```